### ISSUE TITLE
Idiot Proof

### ISSUE THREAD

**Issue by pedromorgan** [revel/revel.github.io#45]
Make the docs more idiot proof..

We need to welcome php and django et all devs..

So sometimes we need to define terms..
eg static.. wtf is static serve? to a php programmer

Also me need to take you/your ouT and more into the/it that...defned..

This will make it more multilingual also.. (I think in Welsh when i code docs = idiots language)

Also we need chinese and nippongo translations..

**Comment by brendensoares** [comment]
@pedromorgan We already have Chinese, Russian and soon Korean translations. I wonder how we can help keep them up to date? @arahansa

**Comment by anonx** [comment]
@brendensoares 
> We already have Chinese, Russian and soon Korean translations. I wonder how we can help keep them up to date? @arahansa

Aaaaand @gorevelru, @goofcc

**Comment by pedromorgan** [comment]
Not sure we can run "jekyll plugins" on github..

but the other way to do would be with a directory for each lang..
en/, fr/ ru/ etc

There again, we'd need to sync than will all the other "participants".

**Comment by anonx** [comment]
> but the other way to do would be with a directory for each lang..
> en/, fr/ ru/ etc

@pedromorgan Do you think the translations should be centralized?

Right now docs are kind of federated. Revel Core Team is responsible for English version only. Translations are driven by community. Moreover, there are kind of separate communities around translated versions. They may have their own forums, moderators, etc. And what's important authors of translations can add ADs to their web-site for monetization.

**Comment by pedromorgan** [comment]
>>anonx:  Do you think the translations should be centralized?

No, i dont think so, and that's from experience with Smarty docs from yonder year, and was complicated by permissions, abandoned users, incorrect translation and hijacking  et all

However we could help by maybe linking to the "other langs" on the sire somehow, maybe with a dropdown on the "top nav right side"..

**Comment by anonx** [comment]
> However we could help by maybe linking to the "other langs" on the site somehow, maybe with a dropdown on the "top nav right side"..

:+1:

**Comment by brendensoares** [comment]
@anonx @pedromorgan I like the idea of having the translations central to this repo and giving other community members access. At the very least, we could use a PR work flow.

**Comment by arahansa** [comment]
@brendensoares  
Ah, I just see this mention. 
http://100coding.com/go/revel 
This site is translated Revel site in Korea.. But All of content is not translated. 
I translated main page("메인") and tutorial page("튜토리얼") so far. 
I was busy last year. sorry. 
hm... I added a button called "번역" in all paragraph.
If visitor push a button "번역", they will see the text changed in English or Korean.
Because I respect the original manual. 

Ah.. I want to ask something... 
As a administrator of Golang Korea facebook group,
I have been developing web board using Go, Revel, and GORP.
And It is almost completed. I plan to make a video tutorial, PPTslide, blog post and Github repository. and share it with facebook group members.
(I refer to booking sample a lot. thanks)

But I have a problem.. When I implement page nation.
This makes an error---> {{.pageInfo.BeginPage - 1}} 
Is it impossible to insert operator in {{ }} ?
hm.. I am originally a Java web developer. 
When I use this statement ${articlePage.beginPage-1} in JSP , this doesn't make a bug. 
but In golang. It doesn't work well..I don't know simple way how to make a link. 
I don't want to make a new variable for this. 
(if it is elementary question, i'm sorry )

**Comment by goofcc** [comment]
@anonx , @brendensoares,Chinese manual translation, I would have taken the time to make and keep track of the latest version of Revel. Don't know what the official has rules to follow for translation? At present, [Chinese manual](http://gorevel.cn/docs/index.html) has a simple version, I hope to finish the translation when the Revel 1.0 release.

**Comment by brendensoares** [comment]
@goofcc @arahansa I am not familiar with translation best practices. What do you recommend as far as defining translation guidelines/rules?

**Comment by jniltinho** [comment]
add folder doc to translation pt-BR like https://github.com/beego/beedoc or https://github.com/macaron-contrib/docs.
We would like to contribute to the project
